examples/example_plugin/example_plugin/static/example_plugin/docs/404.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
```

Comment:
Why does the plugin need to provide a custom 404 page?

Reply:
mkdocs...